Makefile

```diff
@@ -0,0 +1,101 @@
+# Makefile
+include envfile
+PYTHON_EXE = python3
+TOPDIR = $(shell git rev-parse --show-toplevel)
+PYDIRS="ansible"
+VENV = venv_ansible_virl
+VENV_BIN=$(VENV)/bin
+SRC_FILES := $(shell find ansible -name \*.py)
+SPHINX_DEPS := $(shell find docs/src)
+NON_PYTHON_LIBS := $(shell ls | grep -v vmanage)
+
+help: ## Display help
+	@awk -F ':|##' \
+	'/^[^\t].+?:.*?##/ {\
+	printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
+	}' $(MAKEFILE_LIST)
+
+all: clean venv_ansible_virl check test dist ## Setup ansible-virl env and run tests
+
+venv: ## Creates the needed virtual environment.
+	test -d $(VENV) || virtualenv -p $(PYTHON_EXE) $(VENV) $(ARGS)
+
+$(VENV): $(VENV_BIN)/activate ## Build virtual environment
+
+$(VENV_BIN)/activate: requirements.txt test-requirements.txt
+	test -d $(VENV) || virtualenv -p $(PYTHON_EXE) $(VENV)
+	echo "export TOP_DIR=$(TOPDIR)" >> $(VENV_BIN)/activate
+	. $(VENV_BIN)/activate; pip install -U pip; pip install -r requirements.txt -r test-requirements.txt
+
+deps: venv ## Installs the needed dependencies into the virtual environment.
+	$(VENV_BIN)/pip install -U pip
+	$(VENV_BIN)/pip install -r requirements.txt -r test-requirements.txt
+
+dev: deps ## Installs ansible_virl in develop mode.
+	$(VENV_BIN)/pip install -e ./
+
+check-format: $(VENV)/bin/activate ## Check code format
+	@( \
+	set -eu pipefail ; set -x ;\
+	DIFF=`$(VENV)/bin/yapf --style=yapf.ini -d -r *.py $(PYDIRS)` ;\
+	if [ -n "$$DIFF" ] ;\
+	then \
+	echo -e "\nFormatting changes requested:\n" ;\
+	echo "$$DIFF" ;\
+	echo -e "\nRun 'make format' to automatically make changes.\n" ;\
+	exit 1 ;\
+	fi ;\
+	)
+
+format: $(VENV_BIN)/activate ## Format code
+	$(VENV_BIN)/yapf --style=yapf.ini -i -r *.py $(PYDIRS)
+
+pylint: $(VENV_BIN)/activate ## Run pylint
+	$(VENV_BIN)/pylint --output-format=parseable --rcfile .pylintrc *.py $(PYDIRS)
```

Comment:
change to:
$(VENV_BIN)/pylint --output-format=parseable --rcfile .pylintrc *.py $(SRCFILES)